oak_crypto/proto/v1/crypto.proto

```diff
@@ -47,3 +47,17 @@ message AeadEncryptedMessage {
   bytes ciphertext = 1;
   bytes associated_data = 2;
 }
+
+// Envelope containing session keys required to encrypt/decrypt messages within a secure session.
+// Needed to serialize contexts in order to send them ofer an RPC.
+message CryptoContext {
+  // AEAD key and nonce for encrypting/decrypting client requests.
+  bytes request_key = 1;
+  bytes request_base_nonce = 2;
+  uint64 request_sequence_number = 3;
```

Comment:
Yes, but it will be an incorrect serialization logic since deserialization will inexplicity restart the counter (which will not be obvious for other developers)